css-three-column-layout/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: lato;
+  margin: 0;
+  background-color: #1a96da;
+}
+
+h1 {
+  width: 100%;
+}
+
+.container {
+  max-width: 920px;
+  margin: 30px auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  color: white;
+  text-align: center;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+  padding: 0 10px;
+  width: 100%;
+  text-align: center;
+}
```

Comment:
Just `flex-basis` on `column-` classes. Move anything else to another class that you can add to the element in addition to the `column-` class